custom_components/adjustable_bed/support_report.py

```
@@ -0,0 +1,238 @@
+
+_LOGGER = logging.getLogger(__name__)
+
+# Maximum number of log entries to include
+MAX_LOG_ENTRIES = 100
+
+
+async def generate_support_report(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    coordinator: AdjustableBedCoordinator,
+    include_logs: bool = True,
```

Ruff (0.14.13)

Boolean-typed positional argument in function definition

(FBT001)